src/Microsoft.IdentityModel.Tokens/JsonWebKeyConverter.cs

```diff
@@ -311,6 +314,35 @@ internal static bool TryConvertToX509SecurityKey(JsonWebKey webKey, out Security
             return false;
         }
 
+        internal static bool TryConvertToX509SecurityKey(JsonWebKey webKey, IDictionary<JsonWebKey, List<string>> convertKeyInfos, out SecurityKey key)
```

Comment:
If as mentioned above, we can determine which keys were unable to be converted in a different manner, this new api may not be needed.